backend/msgpack/conn.go

```diff
@@ -28,11 +28,15 @@ import (
 	"time"
 
 	"github.com/m3db/m3x/clock"
-	"github.com/m3db/m3x/log"
+
+	"github.com/uber-go/tally"
 )
 
 const (
 	tcpProtocol = "tcp"
+
+	errorMetric     = "errors"
```

Comment:
nit: Given that these are only used in the `newConnectionMetrics`, perhaps move these consts there?